templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/template_schema.json

```diff
@@ -16,8 +16,7 @@
       "title": "Linux image",
       "description": "Select Linux image to use for VM",
       "enum": [
-        "Ubuntu 18.04",
-        "Ubuntu 18.04 Data Science VM"
+        "Ubuntu 22.04 LTS"
```

Comment:
See comment in porter.yaml: You set the default to a non-existing Ubuntu 22 DSVM.